app/src/main/java/com/postpc/tenq/ui/activities/RoomActivity.java

```diff
@@ -53,6 +58,27 @@ protected void onCreate(Bundle savedInstanceState) {
             }
         }
         initRecyclerViewAndLoadPlaylist();
+
+        recorder = new RecorderService(this);
+        if (ActivityCompat.checkSelfPermission(this, Manifest.permission.RECORD_AUDIO) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
I think it should be disabled until we are able to disable the recorder from the UI (with the "turn off sound awareness" toggle from the settings activity, cause we don't want it running in the background during development